intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/YangKeyStmtCheck.java

```diff
@@ -34,10 +34,20 @@ public void performCheck(@NotNull PsiElement element, @NotNull AnnotationHolder
         String key = "";
         String config = "";
         List<String> keys = new ArrayList<>();
+        List<String> allKeys;
         List<String> leafs = new ArrayList<>();
         if (((YangKeyStmt)element).getKeyArgStr()!= null){
-            key = ((YangKeyStmt)element).getKeyArgStr().getText().replaceAll("\"", "");
-            keys = List.of(key.split(" "));
+            key = ((YangKeyStmt)element).getKeyArgStr().getText().replaceAll("\\s+"," ").replaceAll("\"", "");
+            allKeys = List.of(key.split(" "));
+            for (String k : allKeys){
+                if (!keys.contains(k)){
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/util/Set.html#add(E)

i.e. just use add() and check its return value